coa-helm/templates/mysqldb-deployment.yaml

```diff
@@ -42,4 +42,59 @@ spec:
         - name: configuration-claim1
           persistentVolumeClaim:
             claimName: configuration-claim1
+        - name: {{ include "app.fullname" . }}-mysql-startup
+          configMap:
+            name: {{ include "app.fullname" . }}-mysql-startup
+            items:
+              - key: {{ .Values.config.mysql.startup_scripts.create_dbs }}
+                path: {{ .Values.config.mysql.startup_scripts.create_dbs }}
 status: {}
+
+---
+
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    io.kompose.service: mysqldb
+  name: mysqldb
+spec:
+  ports:
+    - name: {{ .Values.db.port | quote }}
+      port: {{ .Values.db.port }}
+      targetPort: {{ .Values.db.port }}
+  selector:
+    io.kompose.service: mysqldb
+status:
+  loadBalancer: {}
+
+---
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "app.fullname" . }}-mysql-startup
```

Comment:
I've been using the convention `<app.fullname>-<service>-config` for ConfigMap names. Can we use that here?

The `-startup` suffix still makes sense above, where you define the volume name. Giving the ConfigMap and the Volume different names makes it ever so slightly easier to parse for a person reading, IMO